### PR TITLE
Fixes several typos and clarifies some explanations in MySQL doc.

### PR DESCRIPTION
**Pull request by ebozduman**
There were some typos in the bucket notification MySQL README.md file. I fixed the typos and also clarified and reworded some of the explanations in the doc. 

## Motivation and Context
Document changes

## How to test this PR?
Read the document.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

File: docs/bucket/notifications/README.md [hunk 1]
```diff
@@ -979,22 +979,24 @@ MINIO_NOTIFY_MYSQL_COMMENT      (sentence)           optionally add a comment to
 
 `dsn_string` is optional, if not specified, the connection information specified by the `host`, `port`, `user`, `password` and `database` parameters are used.
 
-MinIO supports persistent event store. The persistent store will backup events when the MySQL connection goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
+MinIO supports persistent event store. The persistent store will backup events when MySQL connection goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
```
Comment (klauspost) [inline docs/bucket/notifications/README.md]:
```suggestion
MinIO supports persistent event store. The persistent store will backup events if the MySQL connection goes offline and replays them when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
```

File: docs/bucket/notifications/README.md [hunk 2]
```diff
@@ -941,7 +941,7 @@ The MySQL configuration is located in the `notify_mysql` key. Create a configura
 
 ```
 KEY:
-notify_mysql[:name]  publish bucket notifications to MySQL databases
+notify_mysql[:name]  publish bucket notifications to MySQL databases. When multiple MySQL server endpoints are needed, a user specified "name" is added for each configuration, (e.g."notify_mysql:myinstance").
```
Comment (klauspost) [inline docs/bucket/notifications/README.md]:
```suggestion
notify_mysql[:name]  publish bucket notifications to MySQL databases. When multiple MySQL server endpoints are needed, a user specified "name" can be added for each configuration, (e.g."notify_mysql:myinstance").
```

**Review by klauspost — COMMENTED** [review]
Thanks! Some small tweaks.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#9bdff40c-4c93-4685-b9ff-e7ed4e9d94b2) |
| mint-gateway-azure.sh | :x: [more...](#2258bc98-ed41-46ae-a320-9aa97ece8b0c) |
<a name="9bdff40c-4c93-4685-b9ff-e7ed4e9d94b2"></a>
### 8908-5e228a2/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32235
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c7cffab506c2:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 26 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 1 minutes and 4 seconds
(5/13) Running minio-go tests ... FAILED in 14 minutes and 9 seconds
{
  "args": {},
  "duration": 76193,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="2258bc98-ed41-46ae-a320-9aa97ece8b0c"></a>
### 8908-5e228a2/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30207
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ac0df6f7d689:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 59 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 17 seconds
(4/13) Running mc tests ... done in 7 minutes and 9 seconds
(5/13) Running minio-go tests ... FAILED in 21 minutes and 26 seconds
{
  "args": {
    "bucketName": "minio-go-test-whnyq5evuz1v4zfy",
    "objectPrefix": "",
    "recursive": "true"
  },
  "duration": 241,
  "error": "The specified bucket is not valid.",
  "function": "ListObjects(bucketName, objectPrefix, recursive, doneCh)",
  "message": "PutObject 1 call failed",
  "name": "minio-go: testListObjects",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#923de8ab-9860-468b-bf7d-6fbaa0a529be) |
| mint-fs.sh | :x: [more...](#163e6864-c88f-4889-b14e-94146f63c2fd) |
| mint-gateway-azure.sh | :x: [more...](#3260ffc5-db0d-4a44-92bd-4fc061a0602d) |
<a name="3260ffc5-db0d-4a44-92bd-4fc061a0602d"></a>
### 8908-76994a2/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:31844
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8540633d783b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 3 minutes and 4 seconds
(2/13) Running aws-sdk-java tests ... done in 2 seconds
(3/13) Running aws-sdk-ruby tests ... done in 17 seconds
(4/13) Running mc tests ... done in 7 minutes and 12 seconds
(5/13) Running minio-go tests ... FAILED in 21 minutes and 48 seconds
{
  "args": {
    "bucketName": "minio-go-test-6ek355zxxksbsjn5",
    "objectPrefix": "",
    "recursive": "true"
  },
  "duration": 237,
  "error": "The specified bucket is not valid.",
  "function": "ListObjects(bucketName, objectPrefix, recursive, doneCh)",
  "message": "PutObject 1 call failed",
  "name": "minio-go: testListObjects",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="163e6864-c88f-4889-b14e-94146f63c2fd"></a>
### 8908-76994a2/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30768
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b3d77f5a3a98:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 7 seconds
{"name": "awscli", "duration": 304405, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:30768/awscli-mint-test-bucket-9720""}

Executed 0 out of 13 tests successfully.
```
<a name="923de8ab-9860-468b-bf7d-6fbaa0a529be"></a>
### 8908-76994a2/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32360
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1da097ea8571:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 31 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 51 seconds
(5/13) Running minio-go tests ... FAILED in 14 minutes and 56 seconds
{
  "args": {},
  "duration": 121031,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#b221d4a7-8bfc-46ee-b223-52e78da13fa3) |
| mint-fs.sh | :x: [more...](#c6cb2da3-7454-47e4-823d-6a70f589ba7f) |
| mint-dist-xl.sh | :x: [more...](#d6d63b67-7335-47b7-b844-5f6528327f36) |
| mint-gateway-azure.sh | :x: [more...](#8e11ebba-a85a-4fef-970e-3a742e4f8c0c) |
| mint-gateway-nas.sh | :x: [more...](#2611397a-ea07-487b-a3be-f0523571acff) |
<a name="b221d4a7-8bfc-46ee-b223-52e78da13fa3"></a>
### 8908-1f83af7/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:32348
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 2668df68adc7:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 25 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 50 seconds
(5/13) Running minio-go tests ... FAILED in 14 minutes and 27 seconds
{
  "args": {},
  "duration": 81647,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="c6cb2da3-7454-47e4-823d-6a70f589ba7f"></a>
### 8908-1f83af7/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30783
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a389f940fba8:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 9 seconds
{"name": "awscli", "duration": 307238, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:30783/awscli-mint-test-bucket-2357""}

Executed 0 out of 13 tests successfully.
```
<a name="d6d63b67-7335-47b7-b844-5f6528327f36"></a>
### 8908-1f83af7/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31619
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ac6d19d456b8:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 12 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 4 seconds
(4/13) Running mc tests ... done in 42 seconds
(5/13) Running minio-go tests ... done in 10 minutes and 57 seconds
(6/13) Running minio-js tests ... FAILED in 19 seconds
{
  "name": "minio-js",
  "function": "listObjects(bucketName, prefix, recursive)",
  "args": "bucketName:minio-js-test-a5cb7494-5ba2-4e08-96a5-b78c28277b58, recursive:false",
  "duration": 273,
  "status": "FAIL",
  "error": "Error: done() called multiple times at /mint/run/core/minio-js/node_modules/mocha-steps/lib/step.js:67:11 at /mint/run/core/minio-js/test/functional-tests.js:1161:13 at wrapper (node_modules/minio/node_modules/async/dist/async.js:273:20) at iterateeCallback (node_modules/minio/node_modules/async/dist/async.js:418:21) at /mint/run/core/minio-js/node_modules/minio/node_modules/async/dist/async.js:326:20 at /mint/run/core/minio-js/node_modules/minio/dist/main/minio.js:1846:25 at DestroyableTransform.<anonymous> (node_modules/minio/dist/main/minio.js:514:15) at done (node_modules/readable-stream/lib/_stream_transform.js:192:25) at /mint/run/core/minio-js/node_modules/readable-stream/lib/_stream_transform.js:130:7 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:79:7) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 5 out of 13 tests successfully.
```
<a name="2611397a-ea07-487b-a3be-f0523571acff"></a>
### 8908-1f83af7/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30812
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b51eac19b200:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 7 seconds
{"name": "awscli", "duration": 305290, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:30812/awscli-mint-test-bucket-11547""}

Executed 0 out of 13 tests successfully.
```
<a name="8e11ebba-a85a-4fef-970e-3a742e4f8c0c"></a>
### 8908-1f83af7/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:30565
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e17b06d5a2d1:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 35 seconds
{
  "name": "awscli",
  "duration": 2216,
  "function": "aws --endpoint-url http://72.28.97.61:30565 s3api put-object --body /mint/data/datafile-1-kB --bucket \nAn error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it. --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]To see help text, you can run:  aws help  aws <command> help  aws <command> <subcommand> helpUnknown options: occurred, (BucketAlreadyOwnedByYou), when, calling, the, CreateBucket, operation:, Your, previous, request, to, create, the, named, bucket, succeeded, and, you, already, own, it., error"
}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: docs/bucket/notifications/README.md (hunk 1 above)
Comment (ebozduman) [reply docs/bucket/notifications/README.md]: I changed the sentence like below in the light of your comment:
`MinIO supports persistent event store. The persistent store will backup events`**if** `MySQL connection goes offline and`**replay the events**`when the`**broken connection**`comes back`**up**`. `

File: docs/bucket/notifications/README.md (hunk 2 above)
Comment (ebozduman) [reply docs/bucket/notifications/README.md]: OK

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#74093b35-fd5b-44ba-940d-8b8f5eb8324b) |
| mint-large-bucket.sh | :x: [more...](#03b6e4e9-05ef-4392-8f26-7511b9ad9ef0) |
| mint-dist-xl.sh | :x: [more...](#eb6c4650-c754-49aa-8fa4-d8972c2e6b03) |
| mint-gateway-azure.sh | :x: [more...](#4dd8048b-06d4-429a-9a9d-e38fb3014465) |
<a name="03b6e4e9-05ef-4392-8f26-7511b9ad9ef0"></a>
### 8908-8f1f2d8/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:30645
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp cc9e7794397f:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 29 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 51 seconds
(5/13) Running minio-go tests ... FAILED in 14 minutes and 58 seconds
{
  "args": {},
  "duration": 92769,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="4dd8048b-06d4-429a-9a9d-e38fb3014465"></a>
### 8908-8f1f2d8/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32242
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 44067f73f3b9:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 3 minutes and 2 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 16 seconds
(4/13) Running mc tests ... done in 6 minutes and 31 seconds
(5/13) Running minio-go tests ... FAILED in 19 minutes and 41 seconds
{
  "args": {
    "bucketName": "minio-go-test-wotfn4xwdovvjqes",
    "objectPrefix": "",
    "recursive": "true"
  },
  "duration": 234,
  "error": "The specified bucket is not valid.",
  "function": "ListObjects(bucketName, objectPrefix, recursive, doneCh)",
  "message": "PutObject 1 call failed",
  "name": "minio-go: testListObjects",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="74093b35-fd5b-44ba-940d-8b8f5eb8324b"></a>
### 8908-8f1f2d8/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:30458
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp db7b31b5e041:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 9 seconds
{
  "name": "awscli",
  "duration": 7202,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="eb6c4650-c754-49aa-8fa4-d8972c2e6b03"></a>
### 8908-8f1f2d8/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:31133
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c41923532cce:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 16 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 3 seconds
(4/13) Running mc tests ... done in 40 seconds
(5/13) Running minio-go tests ... done in 12 minutes and 15 seconds
(6/13) Running minio-js tests ... FAILED in 18 seconds
{
  "name": "minio-js",
  "function": "listObjects(bucketName, prefix, recursive)",
  "args": "bucketName:minio-js-test-6ee3bc14-a44c-4659-9d77-b92289d21a89, recursive:false",
  "duration": 290,
  "status": "FAIL",
  "error": "Error: done() called multiple times at /mint/run/core/minio-js/node_modules/mocha-steps/lib/step.js:67:11 at /mint/run/core/minio-js/test/functional-tests.js:1161:13 at wrapper (node_modules/minio/node_modules/async/dist/async.js:273:20) at iterateeCallback (node_modules/minio/node_modules/async/dist/async.js:418:21) at /mint/run/core/minio-js/node_modules/minio/node_modules/async/dist/async.js:326:20 at /mint/run/core/minio-js/node_modules/minio/dist/main/minio.js:1846:25 at DestroyableTransform.<anonymous> (node_modules/minio/dist/main/minio.js:514:15) at done (node_modules/readable-stream/lib/_stream_transform.js:192:25) at /mint/run/core/minio-js/node_modules/readable-stream/lib/_stream_transform.js:130:7 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:79:7) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 5 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#de443593-6543-4609-bad4-fb2f06d58080) |
| mint-large-bucket.sh | :x: [more...](#1cdf8eb5-83d9-4ce3-9f3a-b30d73ca9cdb) |
| mint-dist-xl.sh | :x: [more...](#a5a8f25d-b25b-474c-a0bb-2a396c849e92) |
| mint-gateway-azure.sh | :x: [more...](#4e3106cc-3027-4ca0-9354-1710ef64dd16) |
<a name="a5a8f25d-b25b-474c-a0bb-2a396c849e92"></a>
### 8908-cd28279/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32100
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a79f014f341f:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 26 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 3 seconds
(4/13) Running mc tests ... done in 28 seconds
(5/13) Running minio-go tests ... done in 11 minutes and 49 seconds
(6/13) Running minio-js tests ... FAILED in 17 seconds
{
  "name": "minio-js",
  "function": "listObjects(bucketName, prefix, recursive)",
  "args": "bucketName:minio-js-test-1e713a81-b292-47ef-8a7b-db5125592a71, recursive:false",
  "duration": 234,
  "status": "FAIL",
  "error": "Error: done() called multiple times at /mint/run/core/minio-js/node_modules/mocha-steps/lib/step.js:67:11 at /mint/run/core/minio-js/test/functional-tests.js:1161:13 at wrapper (node_modules/minio/node_modules/async/dist/async.js:273:20) at iterateeCallback (node_modules/minio/node_modules/async/dist/async.js:418:21) at /mint/run/core/minio-js/node_modules/minio/node_modules/async/dist/async.js:326:20 at /mint/run/core/minio-js/node_modules/minio/dist/main/minio.js:1846:25 at DestroyableTransform.<anonymous> (node_modules/minio/dist/main/minio.js:514:15) at done (node_modules/readable-stream/lib/_stream_transform.js:192:25) at /mint/run/core/minio-js/node_modules/readable-stream/lib/_stream_transform.js:130:7 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:79:7) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 5 out of 13 tests successfully.
```
<a name="1cdf8eb5-83d9-4ce3-9f3a-b30d73ca9cdb"></a>
### 8908-cd28279/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:32730
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6198f46d04d8:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 22 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 46 seconds
(5/13) Running minio-go tests ... FAILED in 14 minutes and 30 seconds
{
  "args": {},
  "duration": 80570,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="de443593-6543-4609-bad4-fb2f06d58080"></a>
### 8908-cd28279/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:31469
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d107ba1f7e9e:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 11 seconds
{
  "name": "awscli",
  "duration": 8843,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="4e3106cc-3027-4ca0-9354-1710ef64dd16"></a>
### 8908-cd28279/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32240
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 2967ead8c90a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 8 seconds
{"name": "awscli", "duration": 305479, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:32240/awscli-mint-test-bucket-30615""}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by klauspost — APPROVED** [review]

**Review by poornas — APPROVED** [review]

**Pull request merged**